src/accessors/IEnvironmentRead.ts

```diff
@@ -0,0 +1,10 @@
+import { ISettingRead } from './ISettingRead';
+/**
+ * Allows read-access to configuration and other data which is not created by the normal user
+ */
+
+export interface IEnvironmentRead{
```

Comment:
Moved this to a separate interface as during enabling, no transaction data should be accessed, but settings and other config.